tests/unittest/_torch/modules/test_mla_helix.py

```
@@ -0,0 +1,661 @@
+    max_abs_err_idx = [x.item() for x in max_abs_err_idx]
+    max_rel_err_idx = [x.item() for x in max_rel_err_idx]
+    print(
+        f"Rank {rank} {world_size}-GPU: max error index: {max_err_idx} "
+        f"(test/ref values: {values_err}), max abs error index: {max_abs_err_idx} "
+        f"(test/ref values: {values_abs}, err: {max_abs_err}), max rel error index: {max_rel_err_idx} "
+        f"(test/ref values: {values_rel}, err: {max_rel_err}), atol: {atol}, rtol: {rtol}"
+    )
+    isclose = err < atol + rtol * ref_abs
+    ratio_mismatch = 0.0
+    if not isclose.all().item():
+        n_mismatch = (isclose == False).sum().item()
```

Ruff (0.12.2)

Avoid equality comparisons to `False`; use `not isclose:` for false checks

Replace with `not isclose`

(E712)